modular_skyrat/modules/microfusion/code/phase_emitter.dm

```diff
@@ -0,0 +1,204 @@
+////////////////////////////////////////////////////////////// PHASE EMITTERS
+/*
+Basically the heart of the gun, can be upgraded.
+*/
+/obj/item/microfusion_phase_emitter
+	name = "basic microfusion phase emitter"
+	desc = "The core of a microfusion projection weapon, produces the laser."
```

Comment:
desc = "A first-generation phase emitter, this is the core of the weapon and the source of the beam."